source/index.txt

```diff
@@ -95,11 +100,33 @@ What's New
 The :doc:`What's New </whats-new>` section lists new features and
 changes in each version.
 
-Additional Learning
--------------------
+Resources
```

Comment:
I thought you wanted this to be called "Learn" in the team PR review session which the team agreed with.
I think "Resources" omits the objective we want to emphasize ("learn").

It's also unclear what it refers to since it's used in the top level description as well as the Developer Hub one; this doesn't help the reader understand what a resource is and what the sections contain.

Reply:
I changed it here because I saw it being more of a place with additional resources, but I agree, "Learn" is more accurate.